pgmpy/factors/continuous/LinearGaussianCPD.py

```diff
@@ -67,6 +67,14 @@ class LinearGaussianCPD(BaseFactor):
     """
 
     def __init__(self, variable, beta, std, evidence=[]):
+        try:
+            hash(
+                variable
+            )  # It rejects any unhashable type (lists, dicts, sets, custom mutable objects, etc)
+        except TypeError:
+            raise ValueError(
+                f"`variable` argument must be hashable, not {type(variable).__name__}"
```

Comment:
The standard format that we have been using for these messages is: ` .... be hashable. Got {...type..}`